frontend/src/components/Trigger.tsx

```diff
@@ -312,7 +314,7 @@ export default class Trigger extends React.Component<TriggerProps, TriggerState>
           <div>
             {intervalCategory === PeriodicInterval.WEEK && (
```

Comment:
I am wondering if this code will behave as expected if the language is in French. Same hesitancy applies to around line 115 for other enum uses. `type: parsedTrigger.type ?? TriggerType.INTERVALED`

Reply:
@brendangadd